.evg.yml

```diff
@@ -15,6 +15,8 @@ variables:
       - func: run-make
         vars: { target: "${task_name}" }
 
+exec_timeout_secs: 3600 # tasks will time out after 1 hour
```

Comment:
Are these tasks ever going over an hour?

Reply:
It seems safe: looking at the waterfall all the versions I saw finished in ~12 minutes.